sky/engine/core/text/ParagraphBuilder.cpp

```diff
@@ -207,11 +207,11 @@ void ParagraphBuilder::pop()
         m_currentRenderObject = m_currentRenderObject->parent();
 }
 
-void ParagraphBuilder::addText(const String& text)
+void ParagraphBuilder::addText(const char* text)
```

Comment:
fromUTF8 will return a "null" String, which the impl() value of will be a nullptr.  Not sure what RenderText will do at that point.